web/src/regression/core.test.ts

```
@@ -73,9 +74,11 @@ describe('Core functionality regression test suite', () => {
 
     test('2.2.1 User settings are saved and applied', async () => {
         const getSettings = async () => {
-            await driver.page.waitForSelector('.view-line')
+            await retry(async () => {
```

Using `retry` with `waitForSelector()` is somewhat redundant, because `waitForSelector()` basically already polls under the hood. It'll just also hide unexpected errors. If the `waitForSelector()` times out, you can increase the timeout by passing `{ timeout: 123 }`

---

yeah. there is tension between retry and wait timeouts. not sure what's better.  if we want to put these back into CI we need timeouts and not indefinite retries, but for now i'm just trying to get a release out, so i'm making them pass for me.

---

Wait timeouts are better, because `retry` also swallows unexpected errors. FYI `retry` is _not_ indefinite, it retries 10 times by default, so it's mostly equivalent to just increasing the timeout by 10x

---

cool, makes sense

---

removed retry, works without it too